File: 0268-missing-number/0268-missing-number.cpp
class Solution
{
    public:
        int missingNumber(vector<int> &nums)
        {
            int sum = 0;
            int arraySum = 0;
            for (int i = 0; i <= nums.size(); i++)
            {
                sum += i;
                if (i != nums.size())
                {
                    arraySum += nums[i];
                }
            }
            return sum - arraySum;
        }
};

File: 0268-missing-number/0268-missing-number.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        total=0
        arraysum=0
        for i in range(1, len(nums)+1):
            total+=i
            if(i!=len(nums)+1):
                arraysum+=nums[i-1]
        return total-arraysum
            